src/inject/utils/dom.ts

```diff
@@ -97,8 +97,11 @@ export function watchForNodePosition<T extends Node>(
             start = now;
             attempts = 1;
         }
-        if (prevSibling && (prevSibling.parentNode !== parent && prevSibling.parentNode !== null)) {
-            logWarn('Style was moved to another parent, moving node\'s position', node, prevSibling, parent);
+        if (prevSibling.parentNode === null) {
+            stop();
```

Comment:
I'm not sure, but maybe it should `return`.

Reply:
🤦